src/main/groovy/io/seqera/wave/service/builder/DockerBuildStrategy.groovy

```diff
@@ -166,7 +169,10 @@ class DockerBuildStrategy extends BuildStrategy {
             wrapper.add(platform.toString())
         }
 
-        wrapper.add(singularityImage)
+        if(platform.toString().contains("arm64"))
```

Comment:
```suggestion
        if(platform.arch == "arm64")
```

Better to use platform API

Reply:
Also a conditional expression would fit better here

Reply:
done

Reply:
done